keyboards/ep/comsn/hs68/info.json

```diff
@@ -0,0 +1,364 @@
+                    "label": "^",
+                    "x": 6,
+                    "y": 0
+                },
+                {
+                    "label": "&",
+                    "x": 7,
+                    "y": 0
+                },
+                {
+                    "label": "*",
+                    "x": 8,
+                    "y": 0
+                },
+                {
+                    "label": "(",
+                    "x": 9,
+                    "y": 0
+                },
+                {
+                    "label": ")",
+                    "x": 10,
+                    "y": 0
+                },
+                {
+                    "label": "_",
+                    "x": 11,
+                    "y": 0
+                },
+                {
+                    "label": "+",
+                    "x": 12,
+                    "y": 0
+                },
+                {
+                    "label": "Backspace",
+                    "x": 13,
+                    "y": 0,
+                    "w": 2
+                },
+                {
+                    "label": "GRAV",
+                    "x": 15,
+                    "y": 0
+                },
+                {
+                    "label": "Tab",
+                    "x": 0,
+                    "y": 1,
+                    "w": 1.5
+                },
+                {
+                    "label": "Q",
+                    "x": 1.5,
+                    "y": 1
+                },
+                {
+                    "label": "W",
+                    "x": 2.5,
+                    "y": 1
+                },
+                {
+                    "label": "E",
+                    "x": 3.5,
+                    "y": 1
+                },
+                {
+                    "label": "R",
+                    "x": 4.5,
+                    "y": 1
+                },
+                {
+                    "label": "T",
+                    "x": 5.5,
+                    "y": 1
+                },
+                {
+                    "label": "Y",
+                    "x": 6.5,
+                    "y": 1
+                },
+                {
+                    "label": "U",
+                    "x": 7.5,
+                    "y": 1
+                },
+                {
+                    "label": "I",
+                    "x": 8.5,
+                    "y": 1
+                },
+                {
+                    "label": "O",
+                    "x": 9.5,
+                    "y": 1
+                },
+                {
+                    "label": "P",
+                    "x": 10.5,
+                    "y": 1
+                },
+                {
+                    "label": "{",
+                    "x": 11.5,
+                    "y": 1
+                },
+                {
+                    "label": "}",
+                    "x": 12.5,
+                    "y": 1
+                },
+                {
```

Comment:
Enter's JSON object should be moved down to between `~` at Line 224 and `PGUP` at Line 229.